tools/wpt/testfiles.py

```diff
@@ -279,7 +279,7 @@ def affected_testfiles(files_changed,  # type: Iterable[Text]
         nontest_changed_paths.add((full_path, repo_path))
 
     interface_name = lambda x: os.path.splitext(os.path.basename(x))[0]
-    interfaces_changed_names = map(interface_name, interfaces_changed)
+    interfaces_changed_names = [interface_name(interface) for interface in interfaces_changed]
```

Comment:
Getting a flake8 error here because of the untyped lambda, which can probably just move to a member function now that we no longer use map?

tools/wpt/testfiles.py:282: error: Call to untyped function (unknown) in typed context

Reply:
Weird because I don't get any locally with either py27-flake8 or py37-flake8 tox environments. I'll change it anyway. I'm tempted to do everything in the list comprehension line, as a oneliner def won't likely be easier to understand than that.

Reply:
Oops, my bad, the error was actually from mypy, not flake8.